.travis.yml

```diff
@@ -38,4 +45,4 @@ install:
 - scripts/install_test_dependencies.sh
 
 script:
-- scripts/run_test_suite.sh
+- scripts/check_versions.sh
```

Comment:
We used to use Travis, but migrated to CircleCI. We haven't been using Travis for a long time.